Updated example apps; Fixed autorun disable feature

ccuilla opened the pull request:
.versions
   client/startup.js
   examples/flow-router/.meteor/.finished-upgraders
   examples/flow-router/.meteor/packages
   examples/flow-router/.meteor/release
   examples/flow-router/.meteor/versions
   examples/flow-router/package.json
   examples/flow-router/settings.json
   examples/iron-router/.meteor/.finished-upgraders
   examples/iron-router/.meteor/packages
   examples/iron-router/.meteor/release
   examples/iron-router/.meteor/versions
   examples/iron-router/package.json
   examples/iron-router/settings.json
   examples/react-router/.meteor/packages
   examples/react-router/.meteor/release
   examples/react-router/.meteor/versions
   examples/react-router/package.json
   examples/react-router/settings.json
   package.js

ccuilla committed:
Updated example apps; Fixed autorun disable feature

   .versions
   client/startup.js
   examples/flow-router/.meteor/.finished-upgraders
   examples/flow-router/.meteor/packages
   examples/flow-router/.meteor/release
   examples/flow-router/.meteor/versions
   examples/flow-router/package.json
   examples/flow-router/settings.json
   examples/iron-router/.meteor/.finished-upgraders
   examples/iron-router/.meteor/packages
   examples/iron-router/.meteor/release
   examples/iron-router/.meteor/versions
   examples/iron-router/package.json
   examples/iron-router/settings.json
   examples/react-router/.meteor/packages
   examples/react-router/.meteor/release
   examples/react-router/.meteor/versions
   examples/react-router/package.json
   examples/react-router/settings.json
   package.js

cfnelson commented:
👍  Looks good. @ccuilla Did you test it on both? Meteor 1.4.3 is out now as well. Unsure if you want to bump it up again.

ccuilla committed:
Updated example app packages; Added package.json for dev stuff; tweaked npm start script in examples

   examples/flow-router/.meteor/versions
   examples/flow-router/package.json
   examples/iron-router/.meteor/versions
   examples/iron-router/package.json
   examples/react-router/.meteor/versions
   package.json

ghost commented:
Tested on all examples. Updated to latest available versions. Also just tweaked a couple things. New commit coming shortly.

cfnelson reviewed (approved):
👍
